DFC.ServiceTaxonomy.SharedResources/Helpers/Neo4JHelper.cs

```diff
@@ -12,66 +12,103 @@ public class Neo4JHelper
     {
         private const string verificationCypher = "RETURN 1";
 
+        private Neo4jConnection connection;
+        private string graphName = "neo4j";
+
         public IDriver Neo4jDriver { get; private set; }
-        public Neo4JHelper()
+
+        public Neo4JHelper(Neo4jConnection _connection)
         {
+            connection = _connection;
+            if (connection.graphName.Length > 0)
+                graphName = connection.graphName;
+            connect(connection.uri, connection.userName, connection.password);
         }
 
-        public Neo4JHelper ( string uri)
+        private void initalise()
+        {
+            connection = new Neo4jConnection();
+        }
+        public Neo4JHelper ( string _graphName)
         {
-            connect(uri, string.Empty, string.Empty);
+            initalise();
+            if (_graphName.Length > 0)
+                graphName = _graphName;
         }
 
         public Neo4JHelper(string uri, string userName, string password)
         {
+            initalise();
             connect(uri, userName, password);
         }
 
+        public void connect()
```

Comment:
Method names should start with a capital :)

Reply:
thanks for your comment. fixed pushed